README.md

```diff
@@ -0,0 +1,103 @@
+## Generate a new Maven project ##
+
+- Run below command from Command line tool
+
+```text
+mvn archetype:generate -DartifactId=selenium-learning -DgroupId=com.mytest -DarchetypeGroupId=org.apache.maven.archetypes -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4
+```
+
+- Run below command to download Maven Dependencies, compile, and build project
+
+```text
+mvn clean package -e
+```
+OR,
+
+```text
+mvn clean install -e
+```
+**NOTE:** Use argument `-e` to console error messages while building the project 
+
+- Run below command to run unit tests against compile source code using available testing framework
+
+```text
+mvn test
+```
+
+## Add required dependency for Maven project ##
+
+- Update/Modify the deppendencies in `pom.xml`. Refer [Maven Central Repository](https://central.sonatype.com/) to know about the latest version of dependencies.
+
+1. Add Selenium Dependency
+
+```xml
+<dependency>
+      <groupId>org.seleniumhq.selenium</groupId>
+      <artifactId>selenium-java</artifactId>
+      <version>4.18.1</version>
+</dependency>
+```
```

Comment:
**Performance Issue**: The inclusion of Selenium dependency for automated browser interaction suggests significant potential for performance optimization. Selenium, while powerful, can be resource-intensive and slow, particularly in large-scale or complex test suites. This can lead to increased test execution times and potentially delay the feedback loop in development cycles. <br> **Fix**: Consider evaluating alternative approaches or tools that could offer improved performance for browser-based testing. For instance, headless browsers or browser stubs could be utilized for certain types of tests to reduce execution time. Additionally, review the test suite to identify opportunities for parallel test execution, which Selenium supports, to significantly reduce the overall test execution time. <br> **Code Suggestion**: 
 ```
 <dependency>
       <groupId>org.seleniumhq.selenium</groupId>
       <artifactId>selenium-java</artifactId>
 -     <version>4.18.1</version>
 +     <version>LATEST</version>
 </dependency>
 ```